src/main/java/de/chojo/repbot/commands/Roles.java

```diff
@@ -40,6 +46,22 @@ public Roles(DataSource dataSource, ILocalizer loc) {
                                 .add(OptionType.ROLE, "role", "role", true)
                                 .build()
                         )
+                        .add("adddonor", "command.roles.sub.addDonor", argsBuilder()
+                                .add(OptionType.ROLE, "role", "role", true)
+                                .build()
+                        )
+                        .add("addreceiver", "command.roles.sub.addReceiver", argsBuilder()
+                                .add(OptionType.ROLE, "role", "role", true)
+                                .build()
+                        )
+                        .add("removedonor", "command.roles.sub.removeDonor", argsBuilder()
+                                .add(OptionType.ROLE, "role", "role", true)
+                                .build()
+                        )
+                        .add("removereceiver", "command.roles.sub.removeReceiver", argsBuilder()
+                                .add(OptionType.ROLE, "role", "role", true)
+                                .build()
+                        )
```

Comment:
I'd suggest the following for better readability, if it's possible:
```suggestion
                        .add("addDonor", "command.roles.sub.addDonor", argsBuilder()
                                .add(OptionType.ROLE, "role", "role", true)
                                .build()
                        )
                        .add("addReceiver", "command.roles.sub.addReceiver", argsBuilder()
                                .add(OptionType.ROLE, "role", "role", true)
                                .build()
                        )
                        .add("removeDonor", "command.roles.sub.removeDonor", argsBuilder()
                                .add(OptionType.ROLE, "role", "role", true)
                                .build()
                        )
                        .add("removeReceiver", "command.roles.sub.removeReceiver", argsBuilder()
                                .add(OptionType.ROLE, "role", "role", true)
                                .build()
                        )
```

Reply:
Subcommand have to be all lowercase. otherwise the slash commands will break.